self_enrollment_tool/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
This file is not needed

Reply:
This was generated by the django command to start app. will clean it up